### ISSUE TITLE
Suggestion - Ability to select Testing Variation rules

### ISSUE THREAD

**Issue by AaronCreighton** [anhoej/qicharts2#5]
Hi Anhoej, 

I have run the same data with qcc/minitab and qicharts2 and I get slightly different results. Qicharts2 gives slightly narrow band.  I assume this is because qcc(type = "xbar.one") uses Shewart rules where as qicharts2 uses Shewarts rules + Anhoej's rules. 

Qicharts2 I am finding much nicer to use then qcc because of the integration with ggplot.  It would be great to be able to select type in qicharts2 too. Such as "anhoej = true" or type = c(shewart, anhoej, WH).

Simply put, I might have to use qcc because the customer expects Shewarts rules.

Cheers,

Aaron

**Comment by anhoej** [comment]
I cannot answer for minitab, but qicharts2 uses the formulas provided by Montgomery (Introduction to Statistical Process Control, 6th ed.) to calculate control limits. 
The Anhoej rules have nothing to do with control limits. What do you mean by "Shewhart rules"? Do you mean "Western Electric Rules"? Which ones?

Please see the vignette for a deeper explanation on how qicharts2 work: https://anhoej.github.io/qicharts2/articles/qicharts2.html

**Comment by AaronCreighton** [comment]
Hi, I am still relatively new to control charts so apologies if some of my terms are a little off. 

qcc the other quality control charts package in R "creates Shewhat charts".  Which rules Luca Scrucca
 uses seem to be unclear.  The package was last updated in 2017 and reports from earlier versions reference Montgomery 4th edition and Wetherill, G.B. and Brown, D.W. (1991) Statistical Process Control.

Assuming he uses Montgomery, do the rules change from 4th to 6th?

Here is my code (for continue individual samples):

Qicharts2:

`qic(x = dataset$Date, y = dataset$values, chart = "i", xlab="Date",ylab="Individual Value")`

UCL: 25.13    Center: 12.36   LCL:  -0.40

QCC:

`qcc(Values,type="xbar.one",nsigmas=3, labels=dataset$Date, xlab="Date", ylab="Individual Value")`

UCL: 25.4041   Center: 12.36301  LCL: -0.6780721

Minitab:

UCL: 25.4  Center: 12.36  LCL: ?

I will need to look in to minitab to learn what I can about it"s process.

**Comment by anhoej** [comment]
qicharts2 screens moving ranges for unusually large values before calculating the control limits in an I chart. This sometimes gives a little narrower control limits than qcc. My guess is that Minitab doesn't allow negative limits.
